client/src/pages/Home.js

```diff
@@ -50,10 +51,16 @@ const HomepageHeading = ({ mobile }) => (
         marginTop: mobile ? '0.5em' : '1.5em',
       }}
     />
-    <Button primary size='huge'>
-      Start Trading
+    <MyModal
```

Comment:
Hey Rupa & Guy,

This is where I'm wrapping a button with the MyModal component, to give the modal functionality to that button. I tested it with the "Start Trading" button JUST as a test.